File: pset6/mario/mario.py
from cs50 import get_int

# Sets height variable for prompting the user
height = 0

# Keeps prompting the user for a correct height
while height < 1 or height > 8:
    height = get_int("Height: ")

# Draws collumns
for i in range(height):

    # Draws left-side spaces
    for j in range(height - 1, i, -1):
        print(" ", end="")
        j -= 1
    # Draws left-side blocks
    for k in range(i + 1):
        print("#", end="")
        k += 1
    # Draws middle spaces
    print("  ", end="")

    # Draws right-side blocks
    for l in range(i + 1):
        print("#", end="")
        l += 1

    # Prints a new line
    print("")
    i += 1

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt the user for the piramids height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    //prints collumns
    for (int i = 0; i < height; i++)
    {
        //prints piramid rows - left piramid
        for (int j = height - 1; j > i ; j--)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        //prints the interval between piramids
        printf("  ");
        //prints right piramid
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}